File: IT/1/Oggetti/Python/receptionist.py
# file receptionist.py

from __future__ import print_function
if hasattr(__builtins__, 'raw_input'): input = raw_input

print("Benvenuto nell'Hotel!")

n = input("Nome: ")
s = input("Cognome: ")

print("Benvenuto, {} {}!".format(n, s))

File: IT/1/Oggetti/C++/receptionist.cpp
/* file receptionist.cpp */

#include <iostream>

using std::cout;
using std::cin;
using std::endl;
using std::string;

int main()
{
	cout << "Benvenuto nel nostro Hotel!" << endl;

	cout << "Nome: ";
	string n;
	getline(cin, n);

	cout << "Cognome: ";

	string s;
	getline(cin, s);

	cout << "Benvenuto, " << n << " " << s << "!" << endl;
}

